src/widgets/Tabs/VaccineSelect.js

```diff
@@ -162,6 +173,12 @@ const VaccineSelectComponent = ({
           isDisabled={selectedBatches.length === 0 && !hasRefused}
           onPress={confirmPrescription}
         />
+        <PageButton
+          text={generalStrings.ok_and_next}
+          style={{ marginLeft: 5 }}
+          isDisabled={selectedBatches.length === 0}
```

Comment:
I would think that it should be enabled if the patient has refused too

```suggestion
          isDisabled={selectedBatches.length === 0 && !hasRefused}
```

Reply:
Nice, thanks!